gateway/src/lib.rs

```diff
@@ -676,24 +559,12 @@ pub mod tests {
     impl World {
         pub fn context(&self) -> WorldContext {
             WorldContext {
-                docker: self.docker.clone(),
-                container_settings: self.settings.clone(),
                 hyper: self.hyper.clone(),
                 auth_uri: self.auth_uri.clone(),
             }
         }
     }
 
-    impl DockerContext for WorldContext {
```

Comment:
Do we still need `World` and `WorldContext`?

Reply:
I think gateway should be able to follow the same pattern as our other services by having a `Service` with the main logic. But maybe that should be in another PR?

Reply:
I was thinking the testing would be refactored in a follow up, yes